src/test/java/redis/clients/jedis/commands/jedis/ControlCommandsTest.java

```diff
@@ -440,6 +442,24 @@ public void latencyDoctor() {
     assertNotNull(report);
   }
 
+  @Test
+  public void latencyLatest() {
+    Map<String, LatencyLatestInfo> report = jedis.latencyLatest();
+    assertNotNull(report);
+  }
+
+  @Test
+  public void latencyHistoryFork() {
+    List<LatencyHistoryInfo> report = jedis.latencyHistory(Protocol.EventKeyword.FORK);
+    assertNotNull(report);
+  }
+
+  @Test
+  public void latencyReset() {
+    Long report = jedis.latencyReset();
+    assertNotNull(report);
```

Comment:
It's actually `long` in method definition. So it won't be null anyway.
```suggestion
    assertTrue(jedis.latencyReset() >= 0);
```

Reply:
made a change like yours
make it by amend option